assets/src/js/external_links.js

```diff
@@ -10,7 +10,7 @@ export const setupExternalLinks = function($) {
       </g>
   </svg>`;
 
-  $('div.page-template a:not(.btn), article a:not(.btn)').each(function () {
+  $('div.page-template a:not(.btn):not[class^="cover-card"], article a:not(.btn)').each(function () {
```

Comment:
I think I found why tests fail :bulb: the selection should be enclosed inside `()`, similar to `(.btn)`. Now the script fails, causing some follow-up js code not to run at all.

You can also make the selector more explicit, since we only need to catch the header.

eg. `:not([class="cover-card-heading"])`

Reply:
In that case I'll just change it to `:not(.cover-card-heading)` :)